packages/cli/src/commands/init/init.js

```diff
@@ -90,6 +93,90 @@ async function createFromExternalTemplate(
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, Do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        await execa('sudo', ['gem', 'install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
+      } catch (err) {
+        throw new CLIError(
+          'Error occurred while trying to install CocoaPods, please install it manually.',
```

Comment:
idea: maybe paste a link to official documentation?

Reply:
nit: we can add "and run this command once again"

Reply:
What would be the link the documentation?

Btw I just found this https://facebook.github.io/react-native/docs/integration-with-existing-apps#3-install-cocoapods which recommends using `brew` to install cocoapods 😛

Reply:
> nit: we can add "and run this command once again"

Do you mean something like: "Error occurred while trying to install CocoaPods, please run this command again."? If so, is that a good idea? asking because we can't guarantee that this will work on a second time.

Reply:
Just changed this, if this is not it then let me know and I'll change it again.